gallery/src/components/NxBinaryDonutChart/NxBinaryDonutChartPage.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+import NxBinaryDonutChartFullExample from './NxBinaryDonutChartFullExample';
+import NxBinaryDonutChartHalfExample from './NxBinaryDonutChartHalfExample';
+import NxBinaryDonutChartMinimalExample from './NxBinaryDonutChartMinimalExample';
+import NxBinaryDonutChartNoHoleExample from './NxBinaryDonutChartNoHoleExample';
+import NxBinaryDonutChartLargeHoleExample from './NxBinaryDonutChartLargeHoleExample';
+import NxBinaryDonutChartQuarterExample from './NxBinaryDonutChartQuarterExample';
+
+const nxBinaryDonutChartFullExample = require('!!raw-loader!./NxBinaryDonutChartFullExample').default;
+const nxBinaryDonutChartHalfExample = require('!!raw-loader!./NxBinaryDonutChartHalfExample').default;
+const nxBinaryDonutChartMinimalExampleCode = require('!!raw-loader!./NxBinaryDonutChartMinimalExample').default;
+const nxBinaryDonutChartNoHoleExample = require('!!raw-loader!./NxBinaryDonutChartNoHoleExample').default;
+const nxBinaryDonutChartLargeHoleExample = require('!!raw-loader!./NxBinaryDonutChartLargeHoleExample').default;
+const nxBinaryDonutChartQuarterExample = require('!!raw-loader!./NxBinaryDonutChartQuarterExample').default;
+
+const NxBinaryDonutChartPage = () =>
+  <>
+      <GalleryDescriptionTile>
```

Comment:
It seems your editor is set to quad-indent jsx.  There are eslint errors all through here.

Reply:
> It seems your editor is set to quad-indent jsx. There are eslint errors all through here.

Fixed b9f8078b190bac0c253d7a7c363ad492ea6a5587